### PR TITLE
Fix links for Azure DevOps Server users.

### PR DESCRIPTION
**Pull request by sand-head**
This pull request fixes links so that they work when using this extension on-premises, thus closing #66. It also upgrades some packages, changes the API version demand from 3.0 to 5.0 (we were already using version 5.0 of the API; this just makes the extension marketplace aware that this extension should only get installed on TFS versions that support API version 5.0), and cleans things up a bit. I also made the repository filter sort alphabetically, which is something I should've done originally when adding it.

I've tested these changes both in my personal Azure DevOps organization, as well as in an instance of Azure DevOps Server Express 2019.

### CODE REVIEW

File: src/azdo-services.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
+interface CoreMessage {
+  id: string;
+  name: string;
+  locationUrl: string;
+}
+
+export async function getOrganizationUrl(orgName: string) {
+  const resourceAreaUrl = "https://dev.azure.com/_apis/resourceAreas/79134C72-4A58-4B42-976C-04E7115F32BF?accountName=" + orgName;
```
Comment (nshenoy) [inline src/azdo-services.ts]: Won't this fail on-prem?

Comment (sand-head) [reply src/azdo-services.ts]: after testing this in my first commit, it worked as intended online, but it gave me back an incorrect URL on-prem, as you point out. that's why I removed it in that second commit, which instead gets the base URL using the iframe referrer.

**Comment by modestotech** [comment]
@sand-head, When will this me merged into master? We're eager to get the links in Devops Server fixed.

**Comment by sand-head** [comment]
I apologize for not merging and releasing this sooner, I've been pre-occupied with some of my classes. I'll get this out today.

**Comment by modestotech** [comment]
This problem is back since some days back. The fix from 6531aed19900ffac33daa0025636b9db8fd20787 seems to have been overwritten, as the last change to [src/PullRequestTable/PullRequestTable.columns.tsx](https://github.com/Mimeo/VSTSExtension-ActivePullRequests/blob/master/src/PullRequestTable/PullRequestTable.columns.tsx )was several months ago (before the fix was merged in).

**Comment by sand-head** [comment]
@modestotech I'm not seeing anything strange; the fix may have been merged in mid-December but the PR was made back in September, so that is in fact the most recent change. I have been told that it didn't fix the issue completely, however, and we're still tracking the issue in #66. as I mentioned there, I think I might have an idea that would fix the issue for everyone, and I'm going to do my best to get it out today. I apologize for the inconvenience.

**Pull request merged**